### PR TITLE
Update Philosophy / Values Docs

### PR DESCRIPTION
**Pull request by jay-khatri**
## Summary
- minor edits on proxy docs
- as discussed with @et, restructures our values & philosophy docs. very open to feedback.

## How did you test this change?
clicktest

## Are there any deployment considerations?
n/a

### CODE REVIEW

**Review by Vadman97 — APPROVED** [review]

File: docs-content/general/4_company/1_values.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
-Our mission is to support developers (like you) to ship with confidence. We do this by giving you the tools you need to **uncover, resolve, and prevent** issues in your web app.
+## Overview
 
-With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
-
-## We build in public.
+This doc acts as a reference for our values at highlight.io, which includes how we work and we build things. It acts a way for us to understand how we should operate and what the "startup mentality" means to us. If you'd like to learn more about our product philosophy (which is more related to our product) see [this doc](./product-philosophy.md).
 
-We strive to build in public in every way we can. This means sharing our roadmap, product specs, and company strategy. We see this as giving you all the more reason to consider joining us in [building highlight](https://careers.highlight.run).
+Our values below:
```
Comment (et) [inline docs-content/general/4_company/1_values.md:12]: Omit this line. It's not needed.

File: docs-content/general/4_company/1_values.md [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
 updatedAt: 2022-04-15T02:07:22.000Z
 ---
 
-Our mission is to support developers (like you) to ship with confidence. We do this by giving you the tools you need to **uncover, resolve, and prevent** issues in your web app.
+## Overview
 
-With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
-
-## We build in public.
+This doc acts as a reference for our values at highlight.io, which includes how we work and we build things. It acts a way for us to understand how we should operate and what the "startup mentality" means to us. If you'd like to learn more about our product philosophy (which is more related to our product) see [this doc](./product-philosophy.md).
```
Comment (et) [inline docs-content/general/4_company/1_values.md:10]: * The first two sentences are both "acts as a..." can be consolidated into one sentence
* Second setence "It acts a way for us" -> "It acts **as** a way for us"
* "this doc does this" is overly wordy for no reason -- I'm reading this doc so we don't need to name the doc specifically.
* Same with "product philosophy (which is more related to our product)" -- of course "product philosophy" is related to our product 
* Mentioning the "startup mentality"

File: docs-content/general/4_company/1_values.md [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
 
-We strive to build in public in every way we can. This means sharing our roadmap, product specs, and company strategy. We see this as giving you all the more reason to consider joining us in [building highlight](https://careers.highlight.run).
+Our values below:
 
-## We prioritize cohesion.
+## We build in public.
 
-People may think that we're building multiple products (session replay, error monitoring, etc..) but we see it as one. Before we build anything new, we prioritize making it operate seemlessly with everything else e.g. see our [fullstack mapping guide](../../getting-started/2_frontend-backend-mapping.md).
+With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings.
```
Comment (et) [inline docs-content/general/4_company/1_values.md:16]: This isn't orthogonal to the overview. Per what we chatted about, this should be about how we work as a team not a commitment to our customers.

File: docs-content/general/4_company/1_values.md [hunk 4]
```diff
@@ -1,32 +1,29 @@
...
+Our values below:
 
-## We prioritize cohesion.
+## We build in public.
 
-People may think that we're building multiple products (session replay, error monitoring, etc..) but we see it as one. Before we build anything new, we prioritize making it operate seemlessly with everything else e.g. see our [fullstack mapping guide](../../getting-started/2_frontend-backend-mapping.md).
+With everything we build, we maintain an unwavering promise to use open source technologies so that you and your team don't have a dependency on our hosted offerings. 
 
-## We build for today's developer.
+We strive to build in public in every way we can. This means sharing our roadmap, product specs, and company strategy. We see this as giving you all the more reason to consider joining us in [building highlight](https://careers.highlight.run).
```
Comment (et) [inline docs-content/general/4_company/1_values.md:18]: Similar to the above.

File: docs-content/general/4_company/1_values.md [hunk 5]
```diff
@@ -1,32 +1,29 @@
...
 
-If you're building software in today's ecosystem, you probably want to focus on building software. We challenge ourselves to build developer tooling that’s simple, straightforward but powerful if you'd like to dig deeper. highlight.io is built for developers that want to **develop**. Leave the monitoring stuff to us 👍.
 
 ## We execute quickly and fail fast.
 
-Given that most of the things we build are zero to one, there's often no better way to learn than to build. It's hard to predict how something will scale or be interacted w/ without building something and getting early feedback. With this philosophy, however, its easy to ship low quality work, which is why we prefer to "cut scope, not quality".
+Given that most of the things we build are zero to one, there's often no better way to learn than to build. It's hard to predict how something will scale or be interacted w/ without building something and getting early feedback. With this philosophy, however, its easy to ship low quality work. To address this, we always prefer to "cut scope, not quality", i.e. build out a few, very polished modules rather than many half-baked ones.
```
Comment (et) [inline docs-content/general/4_company/1_values.md:23]: The `i.e.` part is unneeded.

File: docs-content/general/4_company/product-philosophy.md [hunk 6]
```diff
@@ -1,11 +1,13 @@
 ---
-title: Our Product Philosophy
+title: Product Philosophy
 slug: product-philosophy
 ---
 
 ## Overview
 
-This doc acts as a reference for our product philosophy at highlight.io, or to be more exact, how we think about what we build (not necessarily how we build it). It acts as a way for our team to prioritize work as well as educating the public on where we're headed. Sharing this type of content aligns with our values to [build in public](./1_values.md#we-build-in-public) and we hope to continue sharing more in this space.
+This doc acts as a reference for our product philosophy at highlight.io, or to be more exact, how we think about what we build (not necessarily how we build it). It acts as a way for our team to prioritize work. If you'd like to learn more about our company values, check [this doc](./1_values.md).
```
Comment (et) [inline docs-content/general/4_company/product-philosophy.md:8]: Omit the "This doc acts..."

File: docs-content/general/4_company/product-philosophy.md [hunk 7]
```diff
@@ -15,6 +17,12 @@ Our product philosophy at highlight.io is centered around the concept of "cohesi
 
 Prior to working on highlight.io, we all worked at several tech companies of varying sizes, and had first-hand experience trying to stitch together numerous tools to reproduce bugs. It wasn't uncommon that we had to do something like: log into Sentry to see a stacktrace, log into Splunk to query logs, and after investigating with even more tools, give up and log in "as the user" to try and reproduce the issue.
 
+People may think that we're building multiple products (session replay, error monitoring, etc..) but we see it as one. To see this in action, see our [fullstack mapping guide](../../getting-started/2_frontend-backend-mapping.md).
+
+## We build for today's developer.
+
+If you're building software in today's ecosystem, you probably want to JUST focus on building software. We challenge ourselves to build developer tooling that’s simple, straightforward and opiniated, but powerful if want to customize your setup. highlight.io is built for developers that want to **develop**. Leave the monitoring stuff to us 👍.
```
Comment (et) [inline docs-content/general/4_company/product-philosophy.md:24]: `opiniated` -> `opinionated`

**Review by et — COMMENTED** [review]
This is an improvement, but the values doc still isn't resonating.
Per what we chatted about, it was supposed to be how we operate, yet two different target audiences (customers and Highlight (current/future) employees) are being spoken to. 

Personally, I think we should delete it as it's current form of being the top level item in the docs page is just noise for the target audience of our docs (the customer).

If we feel strongly that we should keep it, can we move to the [internal-docs directory](https://github.com/highlight/highlight/tree/main/internal-docs)? My hunch is that the motivation is partly for recruiting new employees. If that's the case, the right candidate will be the one that is poking through our repo.

File: docs-content/general/4_company/product-philosophy.md (hunk 7 above)
Comment (et) [reply docs-content/general/4_company/product-philosophy.md:24]: `but powerful if want` -> `but powerful if you want`

**Pull request merged**